tests/bluetooth/tester/src/btp_bap.c

```diff
@@ -95,6 +100,17 @@ static void audio_send_timeout(struct k_work *work);
 K_THREAD_STACK_DEFINE(iso_data_thread_stack_area, ISO_DATA_THREAD_STACK_SIZE);
 static struct k_work_q iso_data_work_q;
 
+static struct bt_bap_broadcast_source *broadcast_source;
```

Comment:
Have you considered splitting this file into the individual roles, or at least broadcast/unicast?

Reply:
Yes, I am considering doing so in the future.